src/main/twirl/views/showOpportunity.scala.html

```diff
@@ -29,65 +29,132 @@
+                    <dl>
+                        <dt class="heading-small">Deadline</dt>
+                        <dd>Apply any time</dd>
+                    </dl>
+                </div>
+
+                <div class="column-half">
+                    <dl>
+                        <dt class="heading-small">Value</dt>
+                        <dd>&pound;@opportunity.value.amount @opportunity.value.unit</dd>
+                    </dl>
+                </div>
+            </aside>
+        </div>
+
+            }
+        <article>
+            <h2 class="heading-large">@s.sectionNumber. @s.title</h2>
+            @s.paragraphs.map { p =>
+                <p class="text">@p</p>
+            }
+        </article>
+
+        <footer class="contain-floats">
+            <nav role="navigation" class="page-navigation">
                 @prevSection.map { s =>
-                    <div>Previous page<br>
-                        <a href="@controllers.routes.OpportunityController.showOpportunity(opportunity.id, Some(s.sectionNumber))">
-                        @s.title
-                        </a></div>
+                <div class="float-left previous">
+
+                    <span class="arrow-button"></span>
+                    <span class="link">
+                        <a href="@controllers.routes.OpportunityController.showOpportunity(opportunity.id, Some(s.sectionNumber))"
+                           title="Navigate to the previous page."
+                           rel="previous">
+                            <span class="page-navigation-previous">Previous page</span>
+                            <span class="page-navigation-description">@s.title</span>
+                        </a>
+
+                    </span>
+                </div>
                 }
+
                 @nextSection.map { s =>
-                    <div>Next page<br>
-                        <a href="@controllers.routes.OpportunityController.showOpportunity(opportunity.id, Some(s.sectionNumber))">
-                        @s.title
-                        </a></div>
+                <div class="float-right next">
+                    <span class="arrow-button"></span>
+                    <span class="link">
+                        <a href="@controllers.routes.OpportunityController.showOpportunity(opportunity.id, Some(s.sectionNumber))"
+                           title="Navigate to the next page."
+                           rel="next">
+                            <span class="page-navigation-next">Next page</span>
+                            <span class="page-navigation-description">@s.title</span>
+                        </a>
+                    </span>
+                </div>
                 }
-            </footer>
-        }
+            </nav>
+        </footer>
+    </div>
+
+    <div class="column-third">
+        <aside role="complementary">
+            <hr class="hr-blue">
+            <h3 class="bold-small">Supporting information</h3>
```

Comment:
Should be "Support", and the class should be different. Should be made consistent with sidebar.scala.html where the fragment is:

```
<hr class="hr-blue">
<h3 class="heading-medium no-top-margin">Support</h3>
<ul class="list spacious">

```

perhaps consider generalising sidebar.scala.html to take a code block for the <li> tags?